frontend/src/components/main-page/groups-tree-page/common/dependencies/visualization/sankey-links.ts

```diff
@@ -62,15 +62,15 @@ export function buildRawLinks(
       The "natural" flow of Events if from producer to consumer:
       (SomeProducer) --> (Event) --> (SomeConsumer)
     */
-    for (const producedEvent of APIsAndEvents.producedEvents) {
+    for (const producedEvent of APIsAndEvents.publishedEvents) {
       result.push({
         type: 'from-service-or-group',
         from: singleServiceOrGroup,
         to: producedEvent,
       });
     }
 
-    for (const consumedEvent of APIsAndEvents.consumedEvents) {
+    for (const consumedEvent of APIsAndEvents.subscribedEvents) {
       result.push({
         type: 'to-service-or-group',
         from: consumedEvent,
```

Comment:
Could you rename `producedEvent` to `publishedEvent`? And the same for the subscribed events?

Reply:
Addressed in [380ba05](https://github.com/osrgroup/msadoc/pull/75/commits/380ba05360e0fcb697ce53874c65ca76dfba6f0e)